README.md

```diff
@@ -1,21 +1,28 @@
 ## LikeHome Hotel Booking - SE 165
 
-- Sandro Sallenbach (PM)
-- Vinny Senthil
-- Minh Phan
-- Dale Seen
-- Yu Ning Hee
-- Lance Ngo
-- Caffrey Zheng
-- Alex Jiang
-- Tien Nguyen
-- Yuta Sugiura
+This project is done in San Jose State University course cmpe 165 spring 2019.
```

Comment:
Abode is a project by students at San Jose State University as part of their SE 165, Spring 2019 class. It emulates a hotel booking site (like hotels.com) with all its features and functions.

The live version will be available soon on: https://www.abode-hotel.com